services/crates/crates-downloads.service.js

```diff
@@ -87,7 +87,7 @@ module.exports = class CratesDownloads extends BaseCratesService {
       case 'dv':
         return json.crate ? json.versions[0].downloads : json.version.downloads
       case 'dr':
-        return json.crate.recent_downloads
+        return json.crate.recent_downloads || 0
```

Comment:
Could you also add a test case for this scenario, and/or a comment on the schema just to keep some in-code context on this scenario? 

Either a unit test targeting the transform function (like the one for the [crates version badges](https://github.com/badges/shields/blob/master/services/crates/crates-version.spec.js)) or a service test with a mocked response from the crates.io API (https://github.com/badges/shields/blob/master/doc/service-tests.md#5-mocking-responses)